pages/user guides/importing_to_excel.md

```diff
@@ -34,42 +34,133 @@ Ie. If your Application has a dependency on a server, you need to have that serv
 
 ### Importing Virtualization Clusters
 
-Tidal Migration uses the following fields to map a virtualization cluster to other resources: _Hypervisor Technology, Servers_. For this field, you must select the property that the column will correspond to, generally this is _Name_.
+Default fields:
+- Hypervisor Technology
+- Servers
+
+There is no required field when importing Virtualization clusters. 
+
+* _Hypervisor Technology_, this field is used to list the technologies you have in your inventory. 
+
+* _Servers_, this field is used to create associations as you import the rest of your inventory, in this instance, _Servers_ refers to the server (host name) hosting the hypervisor.
+
+    {% include note.html content="Make sure your server inventory (when importing virtual servers) contains a field `Cluster` that points back to the Virtualization Cluster it is part of." %}
+
+
 
 ### Importing Servers
 
-Tidal Migrations uses the following fields to map a server to other resources: _FQDN, IP Addresses, Database Instances, Environment, Cluster_. For these fields, you must select the property that the column will correspond to, generally this is _Name_.
+There is no required field when importing Servers. 
+
+Default fields:
+- Host Name (*)
+- FQDN
+- Assigned ID
+- Zone
+- Role
+- RAM Allocated (GB)
+- RAM Used (GB)
+- CPU Count
+- CPU Name
+- Storage Allocated (GB)
+- Storage Used (GB)
+- Description
+- IP Addresses -> Address (*)
+- Database Instances -> Name (*)
+- Environment -> Name (*)
+- Cluster -> Host Name (*)
+- Tags
+- Operating System Version
+- Operating System
+
+
+**(*)** Associative fields.
+* _Host Name_, This field is used to identify the server and to create association with Databases, Applications, and Virtualization Clusters.
+* _Cluster -> Host Name_, If a server is a Virtual Server, this field should match the value from the _Servers_ field in the Virtualization Cluster inventory. The **cluster host name** field is be used create the necessary association with its Virtual Cluster and its physical host.
+* _Database Instances -> Name_, If a server is hosting a Database (or multiple), this field should contain the **name**(s) of such database(s).
+* _Environment -> Name_, This field is used to represent the server's environment.
+
+
 
 ### Importing Databases
 
-Tidal Migrations uses the following fields to map a database instance to other resources: _Servers and Environment_. For these fields, you must select the property that the column will correspond to, generally this is _Name_.
+There is no required field when importing Databases.
+
+Default fields:
+- Name (*)
+- Description
+- Database Size (MB)
+- Database Path
+- Server -> Host Name (*)
+- Environment -> Name (*)
+- Tags
+
+**(*)** Associative fields.
+* _Name_, This field is used to identify and connect a Database to an Application and a Server.
```

Comment:
i do understand why _name_ is here, its used to associated the records. Maybe its just me in knowing both the data modelling and external usage, that I find it being here conflicting. When I read it, part of me sees it saying that when importing database records, this field is actively used to identify and connect the database to other records, but it doesn't do that. And internally I know, its not an 'associatian', its a plain old attribute. but maybe calling it associative is best, cause that's is use, except only when importing the other records? I guess really my confusion is this reads the exact same as the one for environment, but they do behave differently, and I can't from this doc alone discern that.

Reply:
Yeah, I see what you mean, @prees1. Maybe one of the issues is the word "connect". 

What about something like this:
```
_Name_, This field is used to identify your Database and it is helpful when associating a Database with an Application and a Server.
```

Reply:
I get stuck on 'helpful' there, is that helpful for me in someway, if so how? or helpful to someone else, who? or to the software?

I'd take what you have and say why it is helpful _to them_ very clearly:

>_Name_, When you are importing other records, Applications and Servers, and you provide a set of dependent databases with them the _Name_ field for Database Instances is used to identify those databases and associate the databases as dependencies for those other records. This is helpful when you have some existing dependencies, between Databases and Servers or Applications, in your Excel files and want to capture and leverage them.

Reply:
Nice. Thanks a lot 👍